CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+## 0.64.0      2025-01-07
+
+* Support for pganalyze Query Tuning Workbooks
+  - The collector now optionally executes on-demand EXPLAIN ANALYZE queries for
+    the new Query Tuning feature via the new pganalyze.explain_analyze() helper
+  - This helper is owned by a separate user which gets assigned table read
+    permissions, and avoids granting the collector user unnecessary permissions
+    directly
+  - You can opt into this feature by creating the helper function, otherwise
+    this new feature is turned off by default
```

Comment:
Trying to clarify that the Query Tuning can still be used without this feature:
>By creating the helper function you opt into automated query runs through the collector. For high security environments, Query Tuning Workbooks can still be used without this feature by running queries manually.

Reply:
Good point, I'll reword.